common/static/js/src/string_utils.js

```diff
@@ -45,4 +47,278 @@
         return _.template(text, {interpolate: /\{(.+?)\}/g})(values);
     };
     this.interpolate_text = interpolate_text;
+
+    edx.StringUtils = edx.StringUtils || (function() {
```

Comment:
Why is it necessary to copy this code? Given that we only need a solution for LMS (since CMS is covered above), how about putting edx-ui-toolkit/js/utils/string-utils and edx-ui-toolkit/js/utils/html-utils on the window object in lms/static/lms/js/require-config.js (by requiring them after the config section and then putting them on the window object)?